### PR TITLE
citgm: fix for v0.10 support 

### PR DESCRIPTION
**Pull request by TheAlphaNerd**
A small change to the way events is used allows us to supoprt
v0.10.

### CODE REVIEW

File: lib/citgm.js [hunk 1]
```diff
@@ -4,6 +4,12 @@
 var util = require('util');
 var EventEmitter = require('events');
 
+// This is a shim to allow for v0.10 to work
+// TODO remove when we no longer support v0.10
+if (EventEmitter.EventEmitter) {
```
Comment (Fishrock123) [inline lib/citgm.js]: This `if` shouldn't be necessary: https://github.com/nodejs/node/commit/6ed861dd7feea33224d1cac8e64010b45974ae51

Comment (TheAlphaNerd) [inline lib/citgm.js]: thanks for the heads up. fixed in https://github.com/TheAlphaNerd/citgm/commit/51dac61db90d7949489bb13085bc9d747f5ed176

**Comment by jasnell** [comment]
LGTM

**Comment by TheAlphaNerd** [comment]
it is worth noting that the test suite for citgm will pass with this patch once npm is updated to v2 on v0.10

**Comment by rvagg** [comment]
lgtm :shipit:

**Pull request merged**